ui/apps/dashboard/src/components/Navigation/old/UserDropdown.tsx

```diff
@@ -59,72 +48,72 @@ export default function UserDropdown() {
       >
         <DropdownMenuGroup className="p-2">
           <OrganizationDropdownMenuItem
-            icon={RiSettings3Line}
+            icon={<RiSettings3Line className="size-4" />}
             href="/settings/user"
             label="User Settings"
           />
         </DropdownMenuGroup>
         <DropdownMenuGroup className="p-2">
           <OrganizationDropdownMenuItem
-            icon={RiMapPinLine}
+            icon={<RiMapPinLine className="size-4" />}
             href="https://roadmap.inngest.com/roadmap"
             label="Roadmap"
           />
           <OrganizationDropdownMenuItem
-            icon={RiNewspaperLine}
+            icon={<RiNewspaperLine className="size-4" />}
             href="https://roadmap.inngest.com/changelog"
             label="Release Notes"
           />
           <OrganizationDropdownMenuItem
-            icon={RiQuestionLine}
+            icon={<RiQuestionLine className="size-4" />}
             href="/support"
             label="Contact Support"
           />
           <OrganizationDropdownMenuItem
-            icon={SystemStatusIcon}
-            href={status.url}
+            icon={<SystemStatusIcon status={status} className="size-4" />}
+            href={'/'}
             label="Status Page"
           />
         </DropdownMenuGroup>
         <DropdownMenuGroup className="p-2">
-          <OrganizationDropdownMenuItem
-            icon={RiShutDownLine}
-            onSelect={() =>
-              signOut(() =>
-                router.push((process.env.NEXT_PUBLIC_SIGN_IN_PATH || '/sign-in') as Route)
-              )
-            }
-            label="Sign Out"
-          />
+          <SignOutButton>
+            <OrganizationDropdownMenuItem
+              icon={<RiShutDownLine className="size-4" />}
+              label="Sign Out"
+            />
+          </SignOutButton>
         </DropdownMenuGroup>
       </DropdownMenuContent>
     </DropdownMenu>
   );
 }
 
-function OrganizationDropdownMenuItem(props: {
-  icon: ComponentType<{
-    className?: string;
-  }>;
+function OrganizationDropdownMenuItem({
+  icon,
+  label,
+  href,
+  onSelect,
+}: {
+  icon: ReactNode;
```

Comment:
Typings and props munging here did not play well with the refactored status item, so I just side stepped this issue by making theses react nodes for now as this will go away with new nav.